src/Authenticator/MiniLoginForm.php

```diff
@@ -31,7 +31,7 @@ public function getRealMeMiniLoginLink()
         $authMethod = $fields->dataFieldByName('AuthenticationMethod')->getValue();
         $token = $fields->dataFieldByName('SecurityID')->getValue();
         $actionName = $action->getName();
-        $actionValue = _t(LoginForm::class . '.LOGINBUTTON', 'LoginAction');
+        $actionValue = _t(LoginForm::class . '.LOGINBUTTON', 'Log in');
```

Comment:
See the other usages of this same localisation key which all use "Log in" as the text - which is clearly the correct text.